buildtest/cli/history.py

```diff
@@ -102,38 +102,20 @@ def list_build_history(no_header=None, terse=None, pager=None, color=None):
 
     if terse:
 
+        join_list = []
+
+        for key in table.keys():
+            join_list.append(table[key])
+
+        t = [list(i) for i in zip(*join_list)]
```

Comment:
perhaps use more meaningful variables for `t` i guess `transposed_list` i am not sure. For `join_list` perhaps use a different name perhaps `row_entry` or something i think that joined list is an entry or use a more meaningful name